app/controlplane/internal/service/attestation.go

```diff
@@ -169,17 +169,25 @@ func (s *AttestationService) Init(ctx context.Context, req *cpAPI.AttestationSer
 		return nil, handleUseCaseErr(err, s.log)
 	}
 
+	var orgName string
 	// Find the organization
-	org, err := s.orgUseCase.FindByID(ctx, robotAccount.OrgID)
-	if err != nil {
-		return nil, handleUseCaseErr(err, s.log)
+	if org := usercontext.CurrentOrg(ctx); org != nil {
```

Comment:
If all tokens (user and API token) have an org, we can safely use `requireCurrentOrg` function instead. Do chainloop still support robot account tokens?